src/components/Screen.tsx

```diff
@@ -64,11 +64,13 @@ export const InnerScreen = React.forwardRef<View, ScreenProps>(
     // To maintain default behavior of formSheet stack presentation style and to have reasonable
     // defaults for new medium-detent iOS API we need to set defaults here
     const {
-      sheetAllowedDetents = 'large',
-      sheetLargestUndimmedDetent = 'all',
+      sheetAllowedDetents = [1.0],
```

Comment:
The appearance won't change. I'm changing defaults here, because the `typeof sheetAllowedDetents` has been changed and I think this is a breaking change.

I think I can made this not breaking by supporting "the legacy values" in form of `medium`, `large`, `all` on the JS side & simply translating them into numeric values somewhere in `InnerScreen`.